config/schema/fastly.schema.yml

```diff
@@ -11,6 +11,9 @@ fastly.settings:
     purge_method:
       type: string
       label: 'Fastly Purge Method'
+    stale_while_revalidate:
```

Comment:
Should we assume this should be set to the value that may have been previously set for `stale_if_error`? Could write an update hook, if so.

Reply:
This is just the button to enable/disable.